src/components/tree-view/stories/Doc.mdx

```diff
@@ -0,0 +1,130 @@
+import { Canvas, Meta, Story, Source, ArgTypes, Markdown } from '@storybook/blocks';
+import * as Stories from './tree.stories';
+import { TreeView } from '../TreeView';
+import { TreeViewItem } from '../TreeViewItem';
+
+<Meta of={Stories}/>
+
+# Tree View
```

Comment:
I cannot put C as a brother of Threads

https://github.com/user-attachments/assets/fea30a13-87ee-412b-93a6-aa020c163328